src/gluonts/model/seq2seq/_forking_estimator.py

```diff
@@ -133,6 +146,8 @@ def __init__(
         quantile_output: QuantileOutput,
         freq: str,
         prediction_length: int,
+        distr_output: DistributionOutput = GaussianOutput(),
+        sampling: bool = True,
```

Comment:
We probably don't need the "sampling" option here: until now, this model produces forecasts in terms of quantiles, as described by the `quantile_output` argument; what you are doing here is to add the option for a distribution output instead, as specified by `distr_output`. No sampling is involved here in any of the two cases.

The simplest change to the signature here is as follow, I think:

```suggestion
        freq: str,
        prediction_length: int,
        quantile_output: Optional[QuantileOutput] = None,
        distr_output: Optional[DistributionOutput] = None,
```

followed by an assertion that makes sure that one and only one between `quantile_output` and `distr_output` is set (i.e., not None)

Reply:
Hi, Lorenzo. Thank you for your suggestions.
Could you explain the part of line 148 and 149?
Do you require the input specify what quantile/distribution output?
Would it be possible that both quantile/distribution are not None?

Reply:
@YuntianYeAWS no, one and only one should be not None. You can test this with the following snippet, which is essentially the logical XOR

```python
assert distr_output is None != quantiles is None
```